test/e2e/framework/framework.go

```diff
@@ -193,35 +196,18 @@ func (f *Framework) setupNamespace(ctx context.Context) {
 	}, metav1.CreateOptions{})
 	o.Expect(err).NotTo(o.HaveOccurred())
 
-	// Wait for user ServiceAccount.
-	By(fmt.Sprintf("Waiting for ServiceAccount %q in namespace %q.", userSA.Name, userSA.Namespace))
-	ctxUserSa, ctxUserSaCancel := watchtools.ContextWithOptionalTimeout(ctx, serviceAccountWaitTimeout)
-	defer ctxUserSaCancel()
-	userSA, err = WaitForServiceAccount(ctxUserSa, f.KubeAdminClient().CoreV1(), userSA.Namespace, userSA.Name)
+	// Send a TokenRequest for user service account.
+	userTokenRequest, err := f.KubeAdminClient().CoreV1().ServiceAccounts(ns.Name).CreateToken(ctx, userSA.Name, &authenticationv1.TokenRequest{
```

Comment:
I don't think TokenRequest is a good fit here. This is user account (long lived) and practically if I want to debug with the same token as CI I explicitly want it to be shared and stored in the Secret. Please create the token secret manually.